src/main/java/io/lettuce/core/pubsub/RedisPubSubListener.java

```diff
@@ -77,4 +77,14 @@ public interface RedisPubSubListener<K, V> {
      */
     void punsubscribed(K pattern, long count);
 
+    /**
+     * Subscribed to a Shard channel.
+     *
+     * @param shardChannel Shard channel
+     * @param count Subscription count.
+     */
+    default void ssubscribed(K chashardChannelnnel, long count) {
```

Comment:
Please add `@since 7.0` to indicate since which version this method was introduced.

Reply:
done